src/Color.js

```diff
@@ -1,4 +1,5 @@
 const COLORS = require('./util/colors');
+const convert = require('color-convert');
```

Comment:
The same as in `package.json`, the project should go without external dependencies